### PR TITLE
Created the separate function to add comments

### LINKED ISSUE #44: [Comments] There is no populated author fields that is needed when getting comments
**Description:** There is no populated author fields that is needed when getting comments.

**Reproducible:** always.

**Preconditions:**
The user is logged in.

**Steps to reproduce:**
1. Create a comment

**Actual result:**
Image

**Expected result:**
Image

### PR DESCRIPTION
**Pull request by holus-bolus**
Changed the function structure accordingly to requirements

image

now when we create the comment, it has the required structure

image

### CODE REVIEW

File: src/services/comment.js [hunk 1]
```diff
@@ -3,12 +3,40 @@ const Comment = require('~/models/comment')
...
+      text,
+      cooperation: cooperationId,
+      authorRole
+    }
+    let comment = new Comment(commentData)
+    await comment.save()
+    comment = await Comment.findById(comment._id).populate({
+      path: 'author',
+      select: 'firstName lastName'
+    })
```
Comment (YaroslavChuiko) [inline src/services/comment.js]:
```suggestion
const comment = await Comment.create({ author, text, cooperation: cooperationId, authorRole })
```

File: src/services/comment.js [hunk 2]
```diff
@@ -3,12 +3,40 @@ const Comment = require('~/models/comment')
...
+        firstName: comment.author.firstName,
+        lastName: comment.author.lastName
+      },
+      text: comment.text,
+      cooperation: comment.cooperation,
+      createdAt: comment.createdAt,
+      updatedAt: comment.updatedAt
+    }
+
+    return result
```
Comment (YaroslavChuiko) [inline src/services/comment.js]:
```suggestion
    return await Comment.findById(comment._id).select('-authorRole').populate({
      path: 'author',
      select: 'firstName lastName'
    })
```

File: src/services/comment.js [hunk 3]
```diff
@@ -3,12 +3,40 @@ const Comment = require('~/models/comment')
...
   },
 
-  getComments: async (cooperationId, userId) => {
-    return await Comment.find({ cooperation: cooperationId, author: userId }).exec()
+  getComments: async (cooperationId) => {
+    return await Comment.find({ cooperation: cooperationId }).populate({
```
Comment (YaroslavChuiko) [inline src/services/comment.js]:
```suggestion
    return await Comment.find({ cooperation: cooperationId }).select('-authorRole').populate({
```

**Review by YaroslavChuiko — CHANGES_REQUESTED** [review]
it can help

File: src/services/comment.js (hunk 3 above)
Comment (holus-bolus) [reply src/services/comment.js]: Correcrted accordingly to suggestions

**Review by YaroslavChuiko — APPROVED** [review]

**Review by OlhaOvsiienko — APPROVED** [review]

**Review by voloshynvv — APPROVED** [review]

**Review by vlad-khrychov — APPROVED** [review]

**Pull request merged**